### ISSUE TITLE
Multiprocess-safe logger for wok

### ISSUE THREAD

**Issue by jrziviani** [kimchi-project/wok#33]
Python logging system is thread-safe* but not multiprocess-safe. Because serial console, which runs in different process, it's necessary to have a file handler that knows how to deal with processes concurrency when writing to a logfile.

**Comment by jrziviani** [comment]
Sent a patch to review.

To be honest I didn't get any issue regarding this multiprocessing issue. However this commit can avoid hard-to-debug issues in future.

I've run some tests and each process gets the lock before writing, for example:
```
(snip)
lock aquired
[log] MainProcess
lock released
--------------
lock aquired
[log] MainProcess
lock released
--------------
lock aquired
[log] SocketServer-4
lock released
--------------
lock aquired
[log] MainProcess
lock released
--------------
(snip)
```

**Comment by alinefm** [comment]
Fixed by 3aad7ba93d0228650af8c95101addd561140cef7
